pixart.js

```diff
@@ -1 +1,43 @@
+var brush = document.querySelector(".brush");
+var button = document.querySelector("#set-color");
+var colorInput = document.querySelector("#color-field");
+var color = "#1B4370";
 
+function startGame() {
+  button.addEventListener("click", setColorButton);
+  colorInput.addEventListener("keydown", enterColorInput);
+};
+
+enterColorInput = function() {
+  var color = colorInput.value;
```

Comment:
You have already defined color as a global variable, so in this case there is no need to define it locally as well. You could write instead `color = colorInput.value;`